packages/circuit-ui/components/Popover/Popover.stories.tsx

```diff
@@ -27,6 +27,11 @@ export default {
   component: Popover,
   parameters: {
     docs: { page: docs },
+    chromatic: {
+      // Fixes flakiness in UI tests. The popover is positioned asynchronously
+      // so we wait before taking the screenshot.
+      delay: 300,
+    },
```

Comment:
Note: this apparently didn't work and was reverted in https://github.com/sumup-oss/circuit-ui/commit/4334aba339b4f0a8739cef4db8c98b3b5c3ae951